tests/integration/docusaurus/expectations/creating_custom_expectations/expect_column_values_to_only_contain_vowels.py

```diff
@@ -5,17 +5,21 @@
     InvalidExpectationConfigurationError,
 )
 from great_expectations.expectations.regex_based_column_map_expectation import (
-    ColumnMapRegexExpectation,
-    ColumnMapRegexMetricProvider,
+    RegexBasedColumnMapExpectation,
+    RegexColumnMapMetricProvider,
 )
 
 
-class ExpectColumnValuesToOnlyContainVowels(ColumnMapRegexExpectation):
+# <snippet>
+class ExpectColumnValuesToOnlyContainVowelsBasedColumnMap(
```

Comment:
Why the name change?

Reply:
???? bizarre. Unintentional and unknown, thanks for catching 🙇

Reply:
Ah - likely a find and replace with the other name changes :)